cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/ResourceDistributionGoal.java

```diff
@@ -229,15 +288,12 @@ protected void rebalanceForBroker(Broker broker,
                                     ClusterModel clusterModel,
                                     Set<Goal> optimizedGoals,
                                     Set<String> excludedTopics)
-      throws AnalysisInputException, ModelInputException {
+      throws ModelInputException {
```

Comment:
We agreed that we will remove ModelInputException.

Reply:
I believe we agreed to remove `AnalysisInputException` -- not the `ModelInputException`.
But it makes sense to remove it as well.